src/modules/MessageSearch/context/hooks/useSearchStringEffect.ts

```diff
@@ -6,7 +6,7 @@ interface DynamicParams {
 }
 
 interface StaticParams {
-  messageSearchDispatcher: ({ type: string, payload: any }) => void;
+  messageSearchDispatcher: (param: { type: string, payload: any }) => void;
```

Comment:
```suggestion
  messageSearchDispatcher: (props: { type: string, payload: any }) => void;
```

Reply:
In a typical development environment, we might use 'props,' but in the context of developing with React, 'props' feels like a reserved word primarily used within components. Therefore, I've used 'params'

Reply:
Yeah I generally we use `props` for components and `params` for others including hooks like here.

Reply:
ah its minor. I just suggested because the other one is using props.